streams/src/main/java/org/apache/kafka/streams/processor/ProcessorContext.java

```diff
@@ -292,8 +290,7 @@ Cancellable schedule(final Duration interval,
      * (including the currently processed record), i.e., it can be considered a high-watermark.
      * Stream-time is tracked on a per-task basis and is preserved across restarts and during task migration.
      *
-     * <p> Note: this method is not supported for global processors (cf. {@link Topology#addGlobalStore} (...)
-     * and {@link StreamsBuilder#addGlobalStore} (...),
+     * <p> Note: this method is not supported for global processors
```

Comment:
```suggestion
     * <p> Note: this method is not supported for global processors (cf. {@link Topology#addGlobalStore(StoreBuilder, String, Deserializer, Deserializer, String, String, ProcessorSupplier)})
```

Reply:
this will break checkstyle, I revert this change.
due to Topology#addGlobalStore has been removed in this PR.

Reply:
This PR remove the old `addGlobalStore(...)` but we should be able to link to the new `addGlobalStore(...)`